app/src/main/java/com/paris_2/san3a/domain/usecase/GetCurrentLocatedUseCase.kt

```diff
@@ -0,0 +1,21 @@
+package com.paris_2.san3a.domain.usecase
+
+import android.util.Log
+import com.paris_2.san3a.domain.entity.States
+import com.paris_2.san3a.domain.repository.LocationRepository
+
+class GetCurrentLocatedUseCase(private val locationRepository: LocationRepository) {
```

Comment:
This is not the current location of the user, maybe he select any government or any city, so use case name should be refactor